src/sequence/mod.rs

```diff
@@ -533,9 +517,9 @@ mod tests {
         )?;
         replica1.apply_data_op(op2.clone())?;
 
-        // we apply the operations in different order, to verify that doesn't affect the result
-        replica2.apply_data_op(op2)?;
+        // Op's must be applied in the same order a source created them (Source Ordering)
```

Comment:
Isn't this problematic? what happens if the op when broadcasted reaches a replica in different order? is this now a limitation in crdts::List?, I wouldn't have expected it to be.